libs/crypto/keys_manager_impl.cpp

```diff
@@ -118,59 +82,66 @@ namespace iroha {
     return true;
   }
 
-  boost::optional<keypair_t> KeysManagerImpl::loadKeys() {
+  boost::optional<shared_model::crypto::Keypair> KeysManagerImpl::loadKeys() {
     std::string pub_key;
     std::string priv_key;
 
-    if (not loadFile(account_name_ + kPubExt, pub_key)
-        or not loadFile(account_name_ + kPrivExt, priv_key))
+    if (not loadFile(account_name_ + public_key_extension, pub_key)
```

Comment:
Revert that plz, constants should follow [our style guide](https://google.github.io/styleguide/cppguide.html#Constant_Names)
Also prefer not changing things, until that is necessary